src/main/bootstrap/.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea
+.gradle
+build
+*.iml
+out
```

Comment:
`.idea`, `*.iml` and `out` are superfluous

They should be in [personal .gitignore_global file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)

Or provide a very looooooooooooooong list of every files to exclude (eclipse, vscode, vim, emacs, log, jvm dump, etc.)

Reply:
Yes, this small file should be extended further with all possible patterns. This is an idea.

Reply:
My personal task goes to https://github.com/github/gitignore/blob/master/Gradle.gitignore

Reply:
Ok, i'll adjust the script to download and concat some files from that repo, and will remove static `.gitignore.

Reply:
Change script to generate `.gitignore` instead of using static file.